Makefile

```diff
@@ -29,7 +29,7 @@ VERSION_LABEL ?= ${VERSION}
 # The go commands and the minimum Go version that must be used to build the app.
 GO ?= go
 GOFMT ?= $(shell ${GO} env GOROOT)/bin/gofmt
-GO_VERSION_KIALI = 1.17.10
+GO_VERSION_KIALI = 1.17.13
```

Comment:
Is this required for this PR? Why is it necessary to bump up the Go version?

If necessary, then before we merge this, we'll need to make an announcement to the dev community that a bump in Go is being introduced ... we don't like forcing infrastructure/dev tooling updates on developers without giving notice.